src/generate/dayjs.ts

```diff
@@ -115,7 +115,7 @@ const generateConfig: GenerateConfig<Dayjs> = {
           parseNoMatchNotice();
           return null;
         }
-        const date = dayjs(formatText, format).locale(localeStr);
+        const date = dayjs(formatText, format, null, true).locale(localeStr);
```

Comment:
https://day.js.org/docs/en/parse/string-format

seems `dayjs(formatText, format, true)` is better here?